### ISSUE TITLE
Scrollbars on preview panel

### ISSUE THREAD

**Issue by ajc2** [script-8/script-8.github.io#73]
Every time the preview panel reloads, it gets these scrolbars.

![Screenshot of scrollbars](http://kland.smilebasicsource.com/i/hluhe.png)

When you interact with the preview panel, they go away.

Firefox 60.0.1 on Linux Mint

**Comment by grantlouisherman** [comment]
Hello, I am new to the code base and wanted to help out.  Two questions:

1. Is this a good starting ticket?
2. Do you want the scroll bars to be disappearing, do you not want them flashing, should the scroll bars be permanent?

**Comment by ajc2** [comment]
@grantlouisherman I don't want them to appear since the panel is typically in full view

**Comment by grantlouisherman** [comment]
hey @ajc2 I am willing to take it if thats cool.

**Comment by gabrielflorit** [comment]
Hi @grantlouisherman, welcome! I *think* this commit fixed it https://github.com/script-8/script-8.github.io/commit/f53d7074ac9bdbae1266448e8f488b1086c21fe8. I can't reproduce it on Mac Chrome/Mac FF/Mac Safari.

**Comment by gabrielflorit** [comment]
Can either of you @grantlouisherman @ajc2  test on FF Linux?

**Comment by ajc2** [comment]
issue still appears to be present

**Comment by gabrielflorit** [comment]
On ff Linux?

**Comment by grantlouisherman** [comment]
<img width="1192" alt="screen shot 2019-01-28 at 2 16 33 pm" src="https://user-images.githubusercontent.com/5606705/51870133-90a53480-2307-11e9-957f-bb0990f50f87.png">

**Comment by grantlouisherman** [comment]
Thats showing it in run mode and it looked fine to me except for the right slider, but I am on a VM. So I am not sure if this is desired behavior

**Comment by gabrielflorit** [comment]
looks like you were running a lesson, right? so the slider would be there because the lesson part is at the bottom. what happens when you load a cassette, no lesson? and then click to CODE,then click pause. do the scrollbars show up?

**Comment by grantlouisherman** [comment]
I apologize if this is ignorant, but was that change that you mentioned deployed? @gabrielflorit

**Comment by gabrielflorit** [comment]
yes, so it's still broken. good to know, thank you.

**Comment by grantlouisherman** [comment]
Sure, do you want me to take a look or should I look for another ticket?

**Comment by gabrielflorit** [comment]
This is a good one I think. Can you try to fix it by using the FF developer tools? It's strange that it only shows up on Linux.

**Comment by grantlouisherman** [comment]
Yeah I can take a look with FF, please assign to me when you get the chance.

**Comment by gabrielflorit** [comment]
If you issue a PR I'll take a look - and thank you so much for offering to work on this!

**Comment by grantlouisherman** [comment]
hey @gabrielflorit @ajc2 I have a fix locally. What is your guys branching strategy like? Should I branch off of dev? For branch names do you guys use <issue-type>/<shortened-issue-title> ?

**Comment by gabrielflorit** [comment]
@grantlouisherman branch off dev, yes. for branch names i haven't done anything consistent - i suppose we should include the issue number there.

**Comment by grantlouisherman** [comment]
I think you need to give me write access @gabrielflorit . Maybe you can make branch for me and just give me write access for that branch?

**Comment by gabrielflorit** [comment]
Can you submit a PR? If you fork off of dev, you can make your changes there and then submit a pull request.

**Comment by grantlouisherman** [comment]
hey @gabrielflorit https://github.com/script-8/script-8.github.io/pull/156

**Comment by grantlouisherman** [comment]
hey @gabrielflorit https://github.com/script-8/script-8.github.io/pull/163

**Comment by gabrielflorit** [comment]
Thanks @grantlouisherman. I merged the fix. @ajc2 do you still see the issue?

**Comment by grantlouisherman** [comment]
Hey @gabrielflorit should I just look for more issues or would you like to tag more help wanted ones?

**Comment by gabrielflorit** [comment]
@grantlouisherman good question. I tagged some as more help. Thank you!
